ui/components/app/assets/asset-list/asset-list-control-bar/index.scss

```diff
@@ -0,0 +1,12 @@
+.asset-list-control-bar {
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
+  padding-top: 16px;
+  margin-left: 16px;
+  margin-right: 16px;
```

Comment:
I'm not sure if there's a formal name for this, but I call adding margins/padding at the root level of a UI element "implied margin/padding," where the spacing is inferred from the parent. Outside of the parent, these margins and paddings don't have much meaning, so I prefer to allow margins and paddings to be applied via props during the component's placement within the parent, for example:

```
<AssetListControlBar marginLeft={16} marginRight={16} paddingTop={16} />
```

It's also fine to set defaults that can be overridden. In the root `Box` element, we might do something like:

```
<Box className="asset-list-control-bar" ref={controlBarRef} marginLeft={16} marginRight={16} paddingTop={16} {...props} />
```

This allows the component to be used in other places without carrying the implied margin/padding with it.

Reply:
https://github.com/MetaMask/metamask-extension/pull/27084/commits/106cc431ecd16632b350a15318e83dd5b6ac5c38